test/upsert/mzcompose.py

```diff
@@ -34,17 +34,14 @@
             "--orchestrator-process-scratch-directory=/mzdata/source_data",
         ],
         additional_system_parameter_defaults={
-            "upsert_source_disk_default": "true",
+            "disk_cluster_replicas_default": "true",
```

Comment:
It seems weird to me to be toggling this to `true` unconditionally, instead of leaving it up to each test, or running with both true and false?

Reply:
we do toggle this in some tests, but some are just testing with a disk and dont want to have to use `c.override`

Reply:
I don't follow! Disk is still a niche option, so it seems good to have the tests that want to test disk opt in explicitly via `c.override`.

Also, the default for this parameter is (currently) set to true globally—def doesn't seem like we want to be setting it to true both in this test file _and_ in services.py?